### DISCUSSION TITLE
关于日志打印问题

### DISCUSSION THREAD

**Discussion by CSpy163** [Tencent/cherry-markdown#170] [Q&A] [answered]
作者大大辛苦了。

我现在遇到日志打印关不掉的问题，在文档里面也没找到相关配置。
我是用 Html 直接引用模块的方式，来使用 CherryMarkdown 的。

<img width="845" alt="image" src="https://user-images.githubusercontent.com/39793958/159117655-faa9b5b7-dde6-4133-8719-8602c2e983e9.png">

![image](https://user-images.githubusercontent.com/39793958/159117503-d2d8f58e-49ca-4c02-a2db-61edef10cc91.png)

直接设置 window.BUILD_ENV 好像没有效果。

**Comment by CSpy163** [comment]
浏览器里的代码是这个效果。

<img width="784" alt="image" src="https://user-images.githubusercontent.com/39793958/159118026-76e2e74d-b5b5-49da-b7b7-a2a60d549c14.png">

**Comment by lyngai** [comment, marked as answer]
生产模式下引入 cherry-markdown.min.js 即可关闭日志打印。

**Reply by CSpy163** [reply]
我这边尝试了一下，可以了。非常感谢😄
